src/components/common/StepMessage.jsx

```diff
@@ -174,6 +175,226 @@ class StepMessage extends Component {
         );
       break;
 
+
+      case 'email-all':
```

Comment:
StepMessage is a common component. The only code that should be here is repeatable code that would be used on other pages. Unless you know that there are other Pressure pages that will need the same table and button, then please use the other case. This why I showed you the 'other' case, which is meant for you to put custom content.